manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml

```diff
@@ -0,0 +1,9 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
+InstallerType: BURN
```

Comment:
```suggestion
InstallerType: burn
```

Reply:
Should probably be updated here then https://docs.microsoft.com/en-us/windows/package-manager/winget/#supported-installer-formats